From de2325d439b5184b30b9542d1cc91b5d3bed7ec1 Mon Sep 17 00:00:00 2001
From: Marco Morais <marcoaz@amazon.com>
Date: Tue, 21 Dec 2021 11:13:20 -0800
Subject: [PATCH] Fix secure tunneling bug sending from destination to source

---
 source/secure_tunneling.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/source/secure_tunneling.c b/source/secure_tunneling.c
index 71fbc48..96e1d31 100644
--- a/source/secure_tunneling.c
+++ b/source/secure_tunneling.c
@@ -484,7 +484,6 @@ static int s_secure_tunneling_send_data(struct aws_secure_tunnel *secure_tunnel,
             &secure_tunnel->send_data_mutex,
             s_can_send_data_status,
             secure_tunnel);
-        secure_tunnel->can_send_data = false;
         aws_mutex_unlock(&secure_tunnel->send_data_mutex);
 
         size_t bytes_max = new_data.len;
-- 
2.33.0

